src/App.jsx

```diff
@@ -3,31 +3,32 @@ import reactLogo from './assets/react.svg'
 import viteLogo from '/vite.svg'
 import './App.css'
 
+let todoList = [
+  {
+    id: 1,
+    title: "Clean up fish tanks"
+  },
+  {
+    id: 2,
+    title: "Feed fish and snails"
+  },
+  {
+    id: 3,
+    title: "Take medication"
+  }
+];
+
 function App() {
   const [count, setCount] = useState(0)
 
   return (
     <>
-      <div>
-        <a href="https://vite.dev" target="_blank">
-          <img src={viteLogo} className="logo" alt="Vite logo" />
-        </a>
-        <a href="https://react.dev" target="_blank">
-          <img src={reactLogo} className="logo react" alt="React logo" />
-        </a>
-      </div>
-      <h1>Vite + React</h1>
-      <div className="card">
-        <button onClick={() => setCount((count) => count + 1)}>
-          count is {count}
-        </button>
-        <p>
-          Edit <code>src/App.jsx</code> and save to test HMR
-        </p>
-      </div>
-      <p className="read-the-docs">
-        Click on the Vite and React logos to learn more
-      </p>
+      <h1>Todo List</h1>
+        <ul>
+          {todoList.map(function (item) {
```

Comment:
@cricket-sama To make this shorter, you could use the arrow function syntax like this:
```
todolist.map((item) => <li></li>)
```